lib/internal/http2/core.js

```diff
@@ -789,6 +708,23 @@ class Http2Session extends EventEmitter {
     debug(`[${sessionName(type)}] http2session created`);
   }
 
+  setNextStreamID(id) {
+    if (typeof id !== 'number')
+      throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'id', 'number');
+    if (id <= 0 || id > 2 ** 31 - 1)
```

Comment:
Extract `0b111111...` trick to a `const` to avoid repetitive calculation, or does V8 optimise that automatically?

Reply:
I believe v8 optimizes these but it's still worthwhile avoid repetition